packages/devtools_app/lib/src/shared/diagnostics/tree_builder.dart

```diff
@@ -116,11 +117,84 @@ void _setupGrouping(DartObjectNode variable) {
   }
 }
 
+void addChildReferences(
+  DartObjectNode variable,
+  RefNodeType refNodeType,
+) {
+  final ref = variable.ref!;
```

Comment:
nit: just to be double careful here, we should `assert(FeatureFlags.evalAndBrowse)`